documentation/modules/ROOT/pages/configuring-aap.adoc

```diff
@@ -0,0 +1,185 @@
+= Configuring Ansible Automation Platform
```

Comment:
I feel like all these sections should be removed and instead get all these steps in our automation. The lab is about 5G RAN OCP Deployments, configuring AAP IMO is outside the scope. I like the idea of leveraging AAP in the workflow. I'd say let's figure out how we can automate all this stuff so it's ready when the user access the lab.

Reply:
We can definitely automate the steps in the "#configuring-automation-controller" and "#installing-mch-addons" sections. This would include general Automation Controller setup (AutomationController instance with existing subscription loaded, generate authentication tokens), the actual creation of the Ansible project and job template, and patching MultiClusterHub with certain add-ons (which can actually just be included in original MultiClusterHub config in my understanding).

We can then move the OpenShift CRDs applied in the "#adding-supporting-objects" section to "adding-automation.adoc" instead, since this section involves simple OpenShift resources that are critical for the Policy creation.

Would this change make sense?

Reply:
Sounds good to me. The Tower token should be generated in our automation and sent via email. I want to avoid users having to access the hypervisor to run cat or something.